packages/drivers/odsp-driver/src/getFileLink.ts

```diff
@@ -101,6 +102,9 @@ export async function getFileLink(
 	return fileLink;
 }
 
+const mockableGetFileLink = mockify(getFileLink);
+export { mockableGetFileLink as getFileLink };
```

Comment:
You can also inline `mockify` when defining the original function, but for this PR it makes the diff really unreadable.